src/main/java/algorithms/curated170/easy/CountingElementsLinear.java

```diff
@@ -0,0 +1,48 @@
+package algorithms.curated170.easy;
+
+public class CountingElementsLinear {
+    public int countElements(int[] arr) {
+        boolean[] check = new boolean[1002];
+        int count = 0;
+        for (int n : arr) {
+            check[n] = true;
+        }
+        for (int n : arr) {
+            if (check[n + 1]) {
+                count++;
+            }
+        }
+        return count;
+    }
+
```

Comment:
It is quite fine.
![image](https://user-images.githubusercontent.com/60903744/118373517-ffcdf200-b5bf-11eb-9027-bcf6b25437c4.png)